pyproject.toml

```diff
@@ -16,6 +16,7 @@ pytest-black = "^0.3.7"
 pytest-cov = "^2.7"
 pytest-mypy = "^0.3.2"
 requests-mock = "^1.6"
+xlsxwriter = "^1.1.8"
```

Comment:
This isn't a dependency of the client, just the examples. I think it would be better for this to be installed separately for those examples, even than installing as a dev dependency here.

Reply:
Could we just add them as a dev dependency within this pyproject.toml?
Otherwise that is going to get complicated for consumers of the examples.

Reply:
I strongly thing we need to separate the examples and the client. I think both are independently valuable, and serve different audiences. The former is about any users of Snyk, the second is about Python developers. Sometimes that will be the same person, but not always.

I think from your viewpoint the examples are more than just examples of using the client, they are the thing you want to put front and center. For that I think a separate repository would serve that need best. That way we don't need to try and serve the two audiences with the same documentation.

Reply:
Yes. As soon as we have pysnyk published we can refactor/remove this again. For now it needs to stay so the repo is usable with the `examples` included.

Reply:
Works for me. Personally I'd add this as an optional dependency, rather than a required dev dependency.

```
poetry add xlsxwriter --optional
```